### PR TITLE
Removed Optimizely-related code and updated so that the new version of "Create section" dialog appears

### PR DESCRIPTION
**Pull request by TurnerRiley**
This is a followup to the PR found [here](https://github.com/code-dot-org/code-dot-org/pull/40384). The original PR contained updates to the "Create section" dialog for teachers to make it easier to use and more clear which login type they should choose. All the changes are the same as the original PR except that the code pertaining to Optimizely has just been removed since we saw statistically significant results that the newer version of the "Create section" dialog for teachers was leading to a higher rate of 3rd-party login choices (signifying to us that it was clearer to users how to connect their section to a 3rd-party login such as Google Classroom).

## Links
The Optimizely results can be found [here](https://app.optimizely.com/v2/projects/12977480133/results/20190492603/experiments/20213215932?previousView=VARIATIONS).

The spec can be found [here](https://docs.google.com/document/d/1YaC8yPL4QUNshj0zXmCZJ1o-FcGYb1Dexv9J0-acHuM/edit#heading=h.rbimn0e5qvmh).

A description of the original PR with explanations on testing and reasoning can be found [here](https://github.com/code-dot-org/code-dot-org/pull/40384).

### CODE REVIEW

File: apps/i18n/common/en_us.json [hunk 1]
```diff
@@ -41,8 +41,7 @@
   "addStudentsMultipleInstructions": "Type or paste the names of your students, one per line.",
   "addStudentsManageMyOwn": "Create and manage my own list of students",
   "addStudentsSyncThirdParty": "Sync my list of students from an existing classroom section in a third party tool",
-  "addStudentsToSectionInstructions": "Choose how you want to add your students:",
-  "addStudentsToSectionInstructionsUpdated": "How do you want your students to sign in?",
+  "addStudentsToSectionInstructions": "How do you want your students to sign in?",
```
Comment (annaxuphoto) [inline apps/i18n/common/en_us.json]: Are we worried about losing the pre-existing translations by switching around the string keys?

**Review by annaxuphoto — COMMENTED** [review]
One comment about the translations, but yay that the experiment succeeded!

File: apps/i18n/common/en_us.json (hunk 1 above)
Comment (TurnerRiley) [reply apps/i18n/common/en_us.json]: Oh yes ur right thank you! I'll update that

**Comment by annaxuphoto** [comment]
Thanks for reverting the strings, but do we need both the original and updated strings? It seems sufficient to just remove the original strings that we're no longer using.

**Review by annaxuphoto — APPROVED** [review]
thanks for adjusting! sorry for the back and forth

**Comment by TurnerRiley** [comment]
No worries! Thank you for the feedback!

**Pull request merged**